src/demo/index.scss

```diff
@@ -5,6 +5,13 @@ body {
         sans-serif;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
+    
+    /* disable selecting texts */
+    user-select: none;
```

Comment:
Do we need to disable selecting text everywhere? If possible we should make this specific to any elements that cause problems, such as buttons.

Reply:
Yes, we can apply this to certain elements.

I just globally disabled this just because I did not find any use cases for selecting text, but perhaps I am wrong?